native_client/dotnet/DeepSpeechClient/Enums/ErrorCodes.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace DeepSpeechClient.Enums
+{
+    /// <summary>
+    /// Error codes from the native DeepSpeech binary.
+    /// </summary>
+    internal enum ErrorCodes
+    {
+        // OK
+        DS_ERR_OK = 0x0000,
+
+        // Missing informations
+        DS_ERR_NO_MODEL = 0x1000,
+
+        // Invalid parameters
+        DS_ERR_INVALID_ALPHABET = 0x2000,
+        DS_ERR_INVALID_SHAPE = 0x2003,
```

Comment:
@lissyx  I've tried to trigger invalid shape with 0.4.1 model and is not returning `0x2001` is returning `0x2003`, can you take a look?

Reply:
@carlfm01 That's because it's not supposed to return a `DS_ERR_INVALID_SHAPE`, the shapes between 0.4.1 and 0.5.0 are compatible for english model. The behavior is sane, you get `DS_ERR_MODEL_INCOMPATIBLE`, so fix your enum :)